packages/react-strict-dom/src/types/StrictTypes.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict
+ */
+
+export type {
+  StrictAnimation,
+  Animatable as StrictAnimatable
+} from './StrictAnimation';
+export type { StrictElement } from './StrictElement';
+export type {
+  AnimationProgressEventTypes as StrictAnimationProgressEventTypes,
+  AnimationProgressEvent as StrictAnimationProgressEvent,
+  StrictAnimationProgressEventTarget,
+  StrictClipboardEventTarget,
+  StrictFocusEventTarget,
+  StrictInputEventTarget,
+  StrictKeyboardEventTarget,
+  StrictPointerEventTarget,
+  StrictWheelEventTarget,
+  StrictMouseEventTarget,
+  StrictTouchEventTarget,
+  StrictEventTarget
+} from './StrictEventTarget';
+export type { StrictHTMLElement } from './StrictHTMLElement';
+export type {
+  StrictHTMLInputElement,
+  StrictHTMLOptionElement,
+  StrictHTMLCollection,
+  StrictHTMLSelectElement,
+  StrictHTMLTextAreaElement
+} from './StrictHTMLFormElements';
+export type { StrictHTMLImageElement } from './StrictHTMLImageElement';
+export type { StrictNode } from './StrictNode';
+export type { StrictProps } from './StrictProps';
+export type { StrictReactDOMProps } from './StrictReactDOMProps';
+export type { StrictReactDOMAnchorProps } from './StrictReactDOMAnchorProps';
+export type { StrictReactDOMButtonProps } from './StrictReactDOMButtonProps';
+export type { StrictReactDOMImageProps } from './StrictReactDOMImageProps';
+export type { StrictReactDOMInputProps } from './StrictReactDOMInputProps';
+export type { StrictReactDOMLabelProps } from './StrictReactDOMLabelProps';
+export type { StrictReactDOMOptionProps } from './StrictReactDOMOptionProps';
+export type { StrictReactDOMSelectProps } from './StrictReactDOMSelectProps';
+export type { StrictReactDOMTextAreaProps } from './StrictReactDOMTextAreaProps';
```

Comment:
I don't think we need to export these because they can be inferred from the relevant `html.*` component export

Reply:
I added them as a convenience feature and because I'm not sure what the TS equivalent of `React.ElementConfig` is. But, yeah, it's not necessary.